.docker/install-plugin-theme.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# Install plugins
+echo "Installing plugins"
+wp --allow-root plugin install akismet buddypress bbpress jetpack wp-multibyte-patch wordpress-importer polldaddy liveblog wp-super-cache custom-content-width camptix-network-tools email-post-changes tagregator supportflow camptix-pagseguro camptix-payfast-gateway camptix-trustpay camptix-trustcard camptix-mercadopago camptix-kdcpay-gateway campt-indian-payment-gateway camptix
```

Comment:
```suggestion
wp --allow-root plugin install akismet buddypress bbpress  classic-editor jetpack wp-multibyte-patch wordpress-importer polldaddy liveblog wp-super-cache custom-content-width camptix-network-tools email-post-changes tagregator supportflow camptix-pagseguro camptix-payfast-gateway camptix-trustpay camptix-trustcard camptix-mercadopago camptix-kdcpay-gateway campt-indian-payment-gateway camptix
```

`MUST` - I'm seeing this at https://wordcamp.test/wp-admin/network/upgrade.php

```
The following plugins are missing:

    classic-editor/classic-editor.php
    wporg-profiles-wp-activity-notifier/wporg-profiles-wp-activity-notifier.php
    json-rest-api/plugin.php

Please let a WordCamp.org developer know about this message.
```

`wporg-profiles-wp-activity-notifier` comes from the Meta repo. `json-rest-api` can be left out since we'll remove it soon anyway.

Reply:
Added `classic-editor` in https://github.com/WordPress/wordcamp.org/pull/113/commits/41867e7168934f7be3093a8ade78a9c1d3604520